packages/js/src/App.tsx

```diff
@@ -30,10 +33,10 @@ export default function App({ config, survey, closeSurvey, errorHandler }: AppPr
       <Modal
         isOpen={isOpen}
         close={close}
-        placement={config.state.product.placement}
-        darkOverlay={config.state.product.darkOverlay}
-        highlightBorderColor={config.state.product.highlightBorderColor}
-        clickOutside={config.state.product.clickOutsideClose}>
+        placement={placement ? placement : product.placement}
```

Comment:
```ts
placement={survey.placement || product.placement}
```
is easier to read and shorter to write 😊

Reply:
done